Lib/test/test_pyclbr.py

```diff
@@ -220,14 +246,14 @@ def test_others(self):
         # These were once some of the longest modules.
         cm('random', ignore=('Random',))  # from _random import Random as CoreGenerator
         cm('pickle', ignore=('partial', 'PickleBuffer'))
+        with temporary_main_spec():
```

Comment:
Can you move this where it was originally? and re-use `cm` (see the diff itself (original lines were L226-L230))